api/src/controllers/editTopic.js

```diff
@@ -0,0 +1,22 @@
+import slugify from 'slugify';
+import { github } from '../config/index.json';
+import { createPullRequestFromData } from '../utils/githubhandler';
+
+export const editTopics = async (req, res) => {
+  const operation = 'update';
+  const { repo, owner, defaultBranch } = github;
+  const bodyData = JSON.stringify(req.body, null, 2);
+  const topicName = slugify(req.body.name.toLowerCase(), '-');
+  const ref = `refs/heads/editTopic/${topicName}`;
+  const response = await createPullRequestFromData(
+    operation,
+    defaultBranch,
+    repo,
+    owner,
+    req,
```

Comment:
The helper function shouldn't need implementation details from the express req.body. This should probably be removed from the function